Seafile.podspec

```diff
@@ -1,6 +1,6 @@
 Pod::Spec.new do |s|
   s.name             = "Seafile"
-  s.version          = "2.7.9"
+  s.version          = "2.8.0"
```

Comment:
Do not put version change in the same commit of other changes. You should always have a separate commit to bump the version